osu.Game.Tournament/Components/TournamentBeatmapPanel.cs

```diff
@@ -135,25 +135,29 @@ private void matchChanged(ValueChangedEvent<TournamentMatch?> match)
                 match.OldValue.PicksBans.CollectionChanged -= picksBansOnCollectionChanged;
             if (match.NewValue != null)
                 match.NewValue.PicksBans.CollectionChanged += picksBansOnCollectionChanged;
-
-            Scheduler.AddOnce(updateState);
+            updateState();
         }
 
         private void picksBansOnCollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
-            => Scheduler.AddOnce(updateState);
+            => updateState();
```

Comment:
please revert these

Reply:
my bad for merge conflict, I will revert